ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v1/fileinput/fileinput.html

```diff
@@ -42,6 +42,7 @@
            accept="${file.accept}"
            id="${file.id}_widget"
            type="file"
+           aria-label="${file.label.value}"
```

Comment:
rich text wont be supported here in accessibility

Reply:
Screen readers are designed to provide auditory feedback to users with visual impairments, and they can recognize and interpret some HTML tags within certain contexts.

Reply:
> rich text wont be supported here in accessibility

Removed

Reply:
> they can recognize and interpret some HTML tags within certain contexts
only when it is not string. this does not holds true in this context as this is a string.